LanguageFeatures/Inline-classes/syntax_A02_t10.dart

```diff
@@ -20,15 +20,15 @@
 // SharedOptions=--enable-experiment=inline-class
```

Comment:
Can't comment on line 16, so here we go:

We assume the same restrictions about the subtypes of a `final` inline class as the ones we have for regular classes.

Given that `inline class B extends A with M1 .. Mk ...` isn't a thing (which includes the case where `k == 0` and there are no mixins), we only need to consider `inline class B implements A {...}`.

So we require that `B` is also `final` (because it must be `base`, and there is no other way to be `base` for an inline class than being `final`), but it is allowed to declare such a `B` inline class in the same library as `A`.

Outside that library, `inline class D implements A ...` is a compile-time error, just like it is for non-inline classes.

Reply:
Thanks!